vault/raft_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package vault
+
+import (
+	"testing"
+)
+
+func TestFormatDiscoveredAddr(t *testing.T) {
```

Comment:
Would be great if we could get a godoc for this function! Something starting with `// TestFormatDiscoveredAddr` and with a brief description of what it does